cereus/src/main/java/com/cactus/entities/User.java

```diff
@@ -3,15 +3,21 @@
  */
 package com.cactus.entities;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.junit.Ignore;
+
+import java.sql.Timestamp;
+
 /**
  * User Entity
  */
-public class User implements Entity {
+public class User implements Entity{
```

Comment:
As per the TA's comments, it might be worth changing `Entity` into an abstract class if our entities share functionality (for example all ID code is the same across classes). However I'm not sure if this will break Jackson's object mapping. 
Saguaro uses Jackson as well, and I think when I tried to abstract away some stuff it broke some reflection stuff. If that's the case, then it's probably fine to get rid of the `Entity` interface altogether, it's not really useful.

Reply:
I think the Jackson library stuff might break for my stuff as well, so are we using the Entity interface anywhere, or can we just take it out?

Reply:
I think its fine to just take it out.